src/main/java/uk/gov/justice/digital/hmpps/whereabouts/controllers/VideoLinkBookingController.kt

```diff
@@ -215,4 +218,24 @@ class VideoLinkBookingController(
     specification: VideoLinkBookingSearchSpecification
   ): VideoLinkBookingOptions =
     videoLinkBookingOptionsService.findVideoLinkBookingOptions(specification)
+
+  @GetMapping(
+    path = ["/video-link-bookings"],
+    produces = ["text/csv"]
+  )
+  @Operation(
+    summary = "Video Link Bookings",
+    description = "Return details of Video Link Bookings in CSV format. Restrict the response to bookings with a main start time within 'days' of start-date."
+  )
+  fun getVideoLinkBookingsByStartDate(
+    @RequestParam(name = "start-date", required = true)
+    @DateTimeFormat(iso = DateTimeFormat.ISO.DATE)
+    @Parameter(description = "The earliest booking start time for which to return bookings for.", required = true)
+    startDate: LocalDate,
+
+    @RequestParam(name = "days")
+    @Parameter(description = "Return details of bookings occurring within this number of days of start-date")
+    days: Long?
+  ) =
+    videoLinkBookingEventService.getBookingsByStartDateAsCSV(startDate, days ?: 7L)
```

Comment:
the only way of getting this info at the moment is reconstruct from the audit - when we move to storing all associated booking info we could move to using that.